c10/core/Device.h

```diff
@@ -146,6 +146,11 @@ struct C10_API Device final {
     return type_ == DeviceType::CPU;
   }
 
+  /// Return true if the device is of PrivateUse1 type.
+  bool is_private_use1() const noexcept {
```

Comment:
nit: can you just make this `is_privateuse1` to reflect the pattern for other variants of this method?

Reply:
Good advice,I have already modified it,please review it again.

> nit: can you just make this `is_privateuse1` to reflect the pattern for other variants of this method?